lib/rspec/core/example.rb

```diff
@@ -566,12 +567,25 @@ def example_skipped?
         # Records the finished status of the example.
         def record_finished(status, finished_at)
           self.status      = status
-          self.finished_at = finished_at
-          self.run_time    = (finished_at - started_at).to_f
+          set_finished_at(finished_at)
+        end
+
+        # @api private
+        # Ensure that finish_time and run_time have values.
+        def finalize_run_time(clock)
```

Comment:
Better name idea! Thanks.

There does need to be some comment there, though, in order to keep 100% documentation coverage (even though it's api private). At least that's what CI tells me :)